### PR TITLE
VIH-6857 Judge Private Consultation

### PR DESCRIPTION
**Pull request by MichaelKeaneathmcts**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: VideoWeb/VideoWeb.AcceptanceTests/Helpers/Users.cs [hunk 1]
```diff
@@ -29,6 +29,16 @@ public static UserDto GetUserFromDisplayName(List<UserDto> users, string display
...
         }
+        public static UserDto GetUserFromUserType(List<UserDto> users, string userType)
+        {
+            if (users.Any(x => x.UserType.ToString().ToLower().Equals(userType.ToLower())))
+            {
+                return users.First(x => x.UserType.ToString().ToLower().Equals(userType.ToLower()));
+            }
+
+            var usersList = users.Select(x => x.UserType).Aggregate("", (current, UserType) => current + userType + ",");
+            throw new InvalidOperationException($"No user with display name '{userType}' found in the list: '{usersList}'");
```
Comment (robertbennett1998) [inline VideoWeb/VideoWeb.AcceptanceTests/Helpers/Users.cs:40]: Looks like there is a typo in the exception message ;) It should be  $"No user with user type '{userType}' found in the list: '{usersList}'"

File: VideoWeb/VideoWeb.AcceptanceTests/Steps/BrowserSteps.cs [hunk 2]
```diff
@@ -98,7 +98,17 @@ private UserDto GetDefaultParticipant()
         private UserDto GetMatchingDisplayName(string userType)
         {
             if (_c.Test.Users.Count != 0)
-                return Users.GetUserFromDisplayName(_c.Test.Users, userType.Replace(" ", string.Empty));
+                return Users.GetUserFromUserType(_c.Test.Users, userType.Replace(" ", string.Empty));
```
Comment (robertbennett1998) [inline VideoWeb/VideoWeb.AcceptanceTests/Steps/BrowserSteps.cs:101]: GetMatchingDisplayName and GetMatchingUserType have the same body, is this intended? Shouldn't GetMatchingDisplayName call something like GetUserFromDisplayName?

**Review by robertbennett1998 — CHANGES_REQUESTED** [review]

File: VideoWeb/VideoWeb.AcceptanceTests/Helpers/Users.cs (hunk 1 above)
Comment (MichaelKeaneathmcts) [reply VideoWeb/VideoWeb.AcceptanceTests/Helpers/Users.cs]: Thanks, I fixed it,

File: VideoWeb/VideoWeb.AcceptanceTests/Steps/BrowserSteps.cs (hunk 2 above)
Comment (MichaelKeaneathmcts) [reply VideoWeb/VideoWeb.AcceptanceTests/Steps/BrowserSteps.cs]: That's corrected, an error when copying code and revising.

**Review by robertbennett1998 — APPROVED** [review]

**Pull request merged**